scripts/jest/preprocess.react-test-suite.js

```diff
@@ -0,0 +1,25 @@
+const babel = require('babel-core');
+const path = require('path');
+
+const babelOptions = {
```

Comment:
i don't think you need a custom transformer here, the root babelrc will be picked up and will work. as it is jest is only running the files in `__tests__` so there isn't a need to filter again in the babel transformer right?

Reply:
Actually there's no filter. L16 is just left over code. I'll remove it.